tcl/org.testeditor.tcl.dsl.ui.tests/src/org/testeditor/tcl/dsl/ui/tests/editor/DropTargetListenerUpdateModelTest.xtend

```diff
@@ -0,0 +1,441 @@
+package org.testeditor.tcl.dsl.ui.tests.editor
+
+import com.google.common.base.Strings
+import java.nio.file.Files
+import java.nio.file.Paths
+import javax.inject.Inject
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.xtext.serializer.ISerializer
+import org.junit.Before
+import org.junit.Test
+import org.testeditor.aml.AmlModel
+import org.testeditor.aml.Component
+import org.testeditor.aml.ComponentElement
+import org.testeditor.dsl.common.testing.AbstractTest
+import org.testeditor.dsl.common.testing.DslParseHelper
+import org.testeditor.tcl.ComponentTestStepContext
+import org.testeditor.tcl.TclModel
+import org.testeditor.tcl.dsl.TclStandaloneSetup
+import org.testeditor.tcl.dsl.ui.editor.DropTargetXtextEditorListener
+import com.google.common.io.ByteStreams
+
+class DropTargetListenerUpdateModelTest extends AbstractTest {
```

Comment:
This test class does not use the given/when/then pattern on which we agreed upon. Also it contains a lot of helper code which will make it very hard to maintain. This should be streamlined with our existing code base.